101-automation/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "workspaceName": {
+            "value": "GEN-UNIQUE"
+        },
+        "location": {
+            "value": "eastus"
+        },
+        "resourcePermissions": {
```

Comment:
remove this and use a defaultValue in the template

Reply:
Done